packages/cli/package.json

```diff
@@ -33,36 +33,36 @@
     "dependencies": {
         "@fastify/cors": "^9.0.1",
         "@vitejs/plugin-react": "^4.2.1",
-        "archiver": "^6.0.1",
+        "archiver": "^7.0.1",
         "cac": "^6.7.14",
         "conf": "^12.0.0",
         "fast-glob": "^3.3.2",
-        "fastify": "^4.26.1",
+        "fastify": "^4.26.2",
         "glob-to-regexp": "^0.4.1",
         "node-fetch": "^3.3.2",
-        "open": "^10.0.3",
+        "open": "^10.1.0",
         "picocolors": "^1.0.0",
         "prompts": "^2.4.2",
-        "vite": "^4.5.2",
+        "vite": "^4.5.3",
         "vite-plugin-externals": "^0.6.2",
         "zod": "^3.22.4"
     },
     "devDependencies": {
-        "@frontify/eslint-config-typescript": "^0.16.2",
+        "@frontify/eslint-config-basic": "^0.20.5",
```

Comment:
Is there something odd with that version? 
As the CI breaks and also locally (same error on both) i can't run lint in the `packages/cli` and also in the `packages/sidebar-settings`
![Screenshot 2024-04-09 at 13 39 46](https://github.com/Frontify/brand-sdk/assets/9362990/8ec4c0b3-0c2b-40db-b9df-bcb5e77bc0db)

Reply:
Had to update eslint in these packages as well, I just applied the order rules and disabled the rest to avoid potential side-effects.